src/local/butler/reproduce_tool/config.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Configuration helpers for the reproduce tool."""
+
+import os
+
+from base import json_utils
```

Comment:
what's the plan for making this tool more standalone? it seems like importing all these things from CF might make it more difficult.

Reply:
Part of the motivation for moving it into the repo was to make it less standalone and avoid duplicating work when trying to replicate the environment. I'm starting to think the way we have things set up today it will be difficult to use something like pants to bundle the tool, so the idea I'm leaning towards at the moment is something like a single script we can point people to that will ensure dependencies are installed, and execute whatever command is necessary to run the tool.

Admittedly the final way that we're going to ship the tool needs more thought, but I think whatever we end up doing is going to have significant dependencies on ClusterFuzz.